framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -1090,13 +1141,19 @@ public S copy() {
         return analysis.createCopiedStore((S) this);
     }
 
+    @SuppressWarnings("unchecked")
     @Override
     public S leastUpperBound(S other) {
+        if (this.isBottom) return other;
+        if (other != null && other.isBottom) return (S) this;
```

Comment:
Do we need to test the nullness of `other` here? The dataflow code is nullness-type-safe guaranteed by NullnessChecker, isn't it?